src/components/ResponsivePlayerWithLinks/ResponsivePlayerWithLinks.tsx

```diff
@@ -0,0 +1,63 @@
+import React, {useState} from "react";
+import ReactPlayer from "react-player";
+
+import styles from "./styles.module.css";
+
+export const ResponsivePlayerWithLinks = ({url, links}) => {
```

Comment:
Are we going to use this player component in the future? If its near future then its ok otherwise I would remove this